Tests/Runtime/MobiledgeX_RuntimeTests.cs

```diff
@@ -146,36 +156,26 @@ public void GetRestURIException(string orgName, string appName, string appVers)
         }
 
         [Test]
-        [TestCase("MobiledgeX", "PingPong", "2.0")]
-        public void WebSocketTest(string orgName, string appName, string appVers)
-        {
-            integration.orgName = orgName;
-            integration.appName = appName;
-            integration.appVers = appVers;
-            var task = Task.Run(async () =>
-            {
-                return await WebSocketTestHelper(orgName, appName, appVers);
-            });
-            // websocket reply messsage printed in console (Register event and Notification event are being emitted on the server)
-            Debug.Log(task.Result);
-            Assert.IsNotEmpty(task.Result.ToString());
-        }
-
-        [Test]
-        [TestCase("WrongCredentials", "WrongAppName", "WrongAppVersion")]
-        public void WebSocketTestExpectedException(string orgName, string appName, string appVers)
+        [TestCase("MobiledgeX-Samples", "sdktest", "9.0", -44949, 29393)]
+        public void FindCloudletFaliure(string orgName, string appName, string appVers , double longtiude, double latitude)
```

Comment:
Not an error, but possible source of confusion: lat, long is more common that long, lat.

Reply:
Although internal, one can actually require a GPS style data structure to enforce field names. Or perhaps always use a specified parameter name to allow almost any parameter order.

Reply:
Agree with Garner here. We do have the Loc object that you can use

Reply:
I don't really need Loc object since I am using setFallbackLocation.
When it comes to lat long instead of long lat, I agree but this might need to happen in another PR in MobiledgeXIntegration setFallBackLocation().